internal/network/cidr_validations.go

```diff
@@ -35,14 +35,14 @@ func VerifySubnetCIDR(cidrStr string) error {
 	if err != nil {
 		return err
 	}
-	ones, _ := cidr.Mask.Size()
-	if ones < 1 || ones > 25 {
-		return errors.New("Address mask size must be between 1 to 25 and must include at least 128 addresses")
+	ones, bits := cidr.Mask.Size()
+	if ones < 1 || ones > bits-7 {
```

Comment:
what is the `-7`? maybe it should be a const? and consider adding a comment explaining why it's -7

Reply:
The error message below indicates we want at least 128 addresses.  This is (2^7).

Reply:
Will add an explanation